src/connectedk8s/azext_connectedk8s/_precheckutils.py

```diff
@@ -155,6 +155,8 @@ def executing_cluster_diagnostic_checks_job(corev1_api_instance, batchv1_api_ins
             else:
                 continue
 
+        _ = azext_utils.make_cluster_diagnostic_checks_job_log(corev1_api_instance, batchv1_api_instance, helm_client_location, kubectl_client_location, kube_config, kube_context, filepath_with_timestamp, storage_space_available)
```

Comment:
should we rename the method to something more clear? current one seems confusing to me